packages/apps/composer-app/src/layouts/AppLayout.tsx

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { useIdentity } from '@dxos/react-client';
+import { Menubar2, Separator, ProfileMenu } from '@dxos/react-uikit';
+
+import { Main } from '../components';
+
+export const AppLayout = () => {
+  const identity = useIdentity();
+  return (
+    <>
+      <Menubar2>
```

Comment:
this one too

Reply:
[Ditto](https://github.com/dxos/dxos/pull/2204#discussion_r1060192851)